src/js/common.js

```diff
@@ -153,12 +162,43 @@ module.exports.loadLatestAssets = (variant, openjdkImp, releaseType, release, ty
   queryAPI(release, url, openjdkImp, type, errorHandler, handleResponse);
 }
 
+module.exports.getChecksum = url => checksums[url]
+
+module.exports.loadChecksum = (url, message) => {
+  return new Promise((resolve, reject) => {
+    loadUrl(url, response => {
+      try {
+        const reader = new FileReader();
+        // This fires after the blob has been read/loaded.
+        reader.addEventListener('loadend', e => {
```

Comment:
What's an `e`?

Reply:
e is the event once the blob is resolved: like another addEventListener callback